recipe/meta.yaml

```diff
@@ -54,13 +55,46 @@ requirements:
     - python
     - pytorch =*=cuda*
```

Comment:
Using the end of the top-level `requirements` to seed this discussion

There are two approaches we can take:

1. Build everything in one go. Then in each split package install the pieces.
2. Split up the builds and installs by components. Make sure to only do the relevant pieces

It looks like we are taking approach 2 so far, which would mean `requirements` moves into each section

The value of approach 1 is we can do the build in one fell swoop. Then the `outputs` are just copying the right bits into place, which is relatively quick

Not entirely sure which is a better approach for this package and our needs. Just wanted to present both options clearly so we could discuss and decide which we prefer

Reply:
Do we need multiple outputs? What is the value in making separate outputs? I don't see a new source URL, so it would seem that these new modules are coming from the same project.

If the answer is yes, we want multiple outputs, I much prefer the seconda approach (build once, then split).

Reply:
Without multiple outputs, we'd bundle the core `flash-attn` and the two extensions in one package? I suspect most people probably don't need the extensions.

Reply:
> 1. Build everything in one go. Then in each split package install the pieces.

I'll try to switch to this.

Reply:
This is done.